src/itensor.jl

```diff
@@ -1912,7 +1921,7 @@ end
 
 *(As::ITensor...; kwargs...)::ITensor = contract(As...; kwargs...)
 
-function contract!(C::ITensor, A::ITensor, B::ITensor, α::Number, β::Number=0)::ITensor
+function contract!(C::ITensor, A::ITensor, B::ITensor, α::Number; β::Number=0)::ITensor
```

Comment:
Or just revert this to `,`.

Reply:
Whoops, didn't notice that! It's odd becuase I downloaded JuliaFormatter yesterday specifically for this. Ensuring its on latest release and rerunning didn't fix (presumably because as far as its concerned theres nothing wrong with a keyword argument?) but changing it manually and then formatting isn't changing it again.

Pushing now.

Reply:
Yeah, it wouldn't change it back from `;` to `,` since it is valid syntax.